src/load/load/load.py

```diff
@@ -0,0 +1,38 @@
+import logging
+
+import hvac
+
+import load.context
+
+from lib.util import (
+  url_host_port
+)
+from lib.vault import Vault
+
+
+def load_secrets(artsy_project, vault_host, vault_port, secrets_file, kvv2_mount_point):
+  ''' load secrets from  Vault and write them to a file '''
+  logging.debug(f'Loading secrets from {vault_host}:{vault_port} under {kvv2_mount_point}')
+
+  path = f'kubernetes/apps/{artsy_project}/'
+
+  vault_client = Vault(
+    url_host_port(vault_host, vault_port),
+    auth_method='kubernetes',
+#    auth_method='iam',
+    role=artsy_project,
+    kvv2_mount_point=kvv2_mount_point,
+    path=path,
+  )
+
+  keys = vault_client.list()['data']['keys']
+
+  logging.debug(f'fetched keys from Vault: {keys}')
+
+  with open(secrets_file, 'w') as f:
+    for key in keys:
+      try:
+        value = vault_client.get(key)
+        f.write(f"export {key}='{value}'\n")
```

Comment:
Would this be writing our secrets out onto the filesystem of a docker container in Circle CI? I wonder if that's a security issue... in terms of the AWS Batch integration, we would be running the script in a Circle container.

Reply:
Ah so our AWS Batch integration will not use this load function at all, since we'll be loading env from the Batch job definition... so you can ignore that.

Reply:
We do need a separate entrypoint in that case for just fetching the secrets without loading them into env

Reply:
will do.

Reply:
I can probably just use the Vault client directly actually as you do above actually

Reply:
it may work. probably best for me to add tests for Vault lib to validate the interface first.